content/docs/reference/design-docs/stdcm/pathfinding_module/standard_allowance.en.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Standard allowance"
+linkTitle: "5 - Standard allowance"
+weight: 50
+---
+
+
+The STDCM module must be usable with
+[standard allowances]({{< ref "docs/explanation/running_time_calculation/allowances" >}} "marge de régularité").
+The user can set an allowance value, expressed either as a function of
+the run time or the travelled distance. This time must be added to the
+run time, so that it arrives later compared to its fastest possible
+run time.
```

Comment:
I'm not sure if it's correct to say run time or running time here, i have a doubt